### ISSUE TITLE
[BUG] Regression: Rich rendering of console status broken in 10.2.0 and above

### ISSUE THREAD

**Issue by tbabej** [willmcgugan/rich#1244]
Rich rendering of the console status is broken in 10.2.0 and above (broken both on 10.2.0 and 10.2.1, works on 10.1.0)

To reproduce, use this minimal snippet

```python
import time
import rich.console
console = rich.console.Console()

console.print("[green] This text is green [/]")
with console.status("[green] This text should also be green [/]"):
    time.sleep(10)
```

which produces the following result on 10.2.0 and later:

![image](https://user-images.githubusercontent.com/2123868/118725447-f7fb9100-b7fd-11eb-89d5-06be182a4d6d.png)

**Comment by willmcgugan** [comment]
Will have a fix in the next version. In the meantime, you can replace that string with `Text.from_markup("[green] green text")`

**Comment by willmcgugan** [comment]
Fixed in 10.2.2
